src/test/scala/org/ergoplatform/mining/ErgoMinerSpec.scala

```diff
@@ -99,7 +99,9 @@ class ErgoMinerSpec extends FlatSpec with ErgoTestHelpers with ValidBlocksGenera
 
         val unsignedTx = new UnsignedErgoTransaction(inputs, outputs)
         val tx = prover.sign(unsignedTx, IndexedSeq(boxToSend), ErgoStateContext(r.h.fullBlockHeight, r.s.rootHash)).get
-        nodeViewHolderRef ! LocallyGeneratedTransaction[ErgoTransaction](tx)
```

Comment:
why?

Reply:
@catena2w, Old approach led to rediculously long logs in Travis so it aborts tests exucution.